src/dodal/devices/pressure_jump_cell.py

```diff
@@ -0,0 +1,239 @@
+from ophyd_async.core import ConfigSignal, StandardReadable
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_rw
+
+
+class PressureJumpCellPumpMode:
+    MANUAL = "Manual"
+    AUTO_PRESSURE = "Auto Pressure"
+    AUTO_POSITION = "Auto Position"
+
+
+class PressureJumpCellBusyStatus:
+    IDLE = "Idle"
+    BUSY = "Busy"
+
+
+class PressureJumpCellTimerState:
+    TIMEOUT = "TIMEOUT"
+    COUNTDOWN = "COUNTDOWN"
+
+
+class PressureJumpCellStopValue:
+    CONTINUE = "CONTINUE"
+    STOP = "STOP"
+
+
+class PressureJumpCellValveControlRequest:
+    OPEN = "Open"
+    CLOSE = "Close"
+    RESET = "Reset"
+    ARM = "Arm"
+    DISARM = "Disarm"
+    # TODO the nones may not be required but FVST and SXST set
+    NONE1 = ""
+    NONE2 = ""
```

Comment:
Should: They're not required. In fact they don't really make sense if you read `""` from the device is that a `NONE1` or `NONE2`? What do they each mean?

Reply:
Removed in 07522bbdf0753fb23ad00510b9f01cee0517c3d3